src/Content/content.stories.mdx

```diff
@@ -1,28 +1,39 @@
-import { Meta, Story, Preview } from "@storybook/addon-docs/blocks";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
 import Content from "./index";
+import Box from "../Box";
 
 <Meta title="Content" component={Content} />
 
+# API
+
+<Props of={Content} />
+
 # Default
 
+Content uses the `foreground` color for sizes default and small, and `white` for size full. You can optionally set the background color with the `bg` prop.
+
 <Preview>
   <Story name="Default">
-    <Content className="bg-foreground" />
+    <Content><Box>Content</Box></Content>
```

Comment:
```suggestion
    <Content>
      <Box>Content</Box>
    </Content>
```
Would be great having it like this for greater visibility, especially being in the docs.